frontend/packages/integration-tests-cypress/tests/app/resource-log.cy.ts

```diff
@@ -0,0 +1,66 @@
+import { checkErrors, testName } from '../../support';
+import { detailsPage } from '../../views/details-page';
+import { errorMessage } from '../../views/form';
+import { listPage } from '../../views/list-page';
+import * as yamlEditor from '../../views/yaml-editor';
+
+const POD_NAME = `pod1`;
+const CONTAINER_NAME = `container1`;
+const testPod = `apiVersion: v1
+kind: Pod
+metadata:
+  name: ${POD_NAME}
+  labels:
+    app: httpd
+spec:
+  securityContext:
+    runAsNonRoot: true
+    seccompProfile:
+      type: RuntimeDefault
+  containers:
+    - name: ${CONTAINER_NAME}
+      image: 'image-registry.openshift-image-registry.svc:5000/openshift/httpd:latest'
+      ports:
+        - containerPort: 8080
+      securityContext:
+        allowPrivilegeEscalation: false
+        capabilities:
+          drop:
+            - ALL`;
+
+describe('Pod log viewer tab', () => {
+  before(() => {
+    cy.login();
+    cy.createProjectWithCLI(testName);
+  });
+
+  afterEach(() => {
+    checkErrors();
+  });
+
+  after(() => {
+    cy.visit('/');
+    cy.deleteProjectWithCLI(testName);
+  });
+
+  it('Create test pod', () => {
+    cy.visit(`/k8s/ns/${testName}/import`);
+    yamlEditor.isImportLoaded();
+    yamlEditor.setEditorContent(testPod).then(() => {
+      yamlEditor.clickSaveCreateButton();
+      cy.get(errorMessage).should('not.exist');
+      detailsPage.sectionHeaderShouldExist('Pod details');
+    });
+  });
+
+  it('Open logs from pod details page tab', () => {
+    cy.visit(`/k8s/ns/${testName}/pods`);
+    listPage.rows.shouldExist(POD_NAME);
+    cy.visit(`/k8s/ns/${testName}/pods/${POD_NAME}`);
+    detailsPage.isLoaded();
+    detailsPage.selectTab('Logs');
+    detailsPage.isLoaded();
+    cy.byTestID('popover-show-full-log').check();
+    cy.byTestID('popover-show-full-log').uncheck();
```

Comment:
> Add check that the number of lines changes

@rhamilto The test pod log is 18 lines and checking for that seems meaningless to me e.g.
`    cy.byTestID('no-log-lines').contains('18 lines');`. 
If we can get a test pod that have over a 1000 lines, that might make sense for a follow-on PR. WDYT?

Reply:
Rather than creating a pod, you could test an existing pod.  Maybe a `kube-apiserver` one?  It seems like you could use the following URL and pick the first one in the table:  `/k8s/ns/openshift-kube-apiserver/core~v1~Pod?name=kube-apiserver-ip-&rowFilter-pod-status=Running&orderBy=desc&sortBy=Owner`

Reply:
> Rather than creating a pod, you could test an existing pod. Maybe a `kube-apiserver` one? It seems like you could use the following URL and pick the first one in the table: `/k8s/ns/openshift-kube-apiserver/core~v1~Pod?name=kube-apiserver-ip-&rowFilter-pod-status=Running&orderBy=desc&sortBy=Owner`

Hmm, actually I thought of this earlier but my understanding of static pod like `kube-apiserver-xxx`  pod name  does change every time a cluster is created. However picking the first one make sense, but I see 0 log line for some of these static pod at times, although I don't know the reason .See attached
![Screenshot 2023-11-28 at 4 27 24 PM](https://github.com/openshift/console/assets/15249132/80e8bc30-a8a2-4124-9b6a-065fd98a35d2)

This could be problematic for the test.